ci/cpu/cuml/build_cuml.sh

```diff
@@ -4,12 +4,11 @@ set -e
 
 if [ "$BUILD_CUML" == '1' ]; then
   echo "Building cuML"
-  CUDA_REL=${CUDA:0:3}
+  CUDA_REL=${CUDA_VERSION%.*}
 
   # Only CUDA 10 supports multigpu ols/tsvd, need to separate the conda build command
-  if [ "${CUDA:0:2}" == '10' ]; then
+  if [ "${CUDA_REL%.*}" == '10' ]; then
```

Comment:
Wouldn't this just be `"${CUDA_REL}" == "10"`

Reply:
CUDA_VERSION seems to follow semantic versioning scheme with three parts, major, minor, patch separated by a dot. The first CUDA_REL=${CUDA_VERSION%.*} strips the version to major and minor versions, where as in this check we strip once more to only compare major versions.